nms/app/components/GatewayCommandFields.tsx

```diff
@@ -130,17 +123,18 @@ function RebootButton(props: ChildProps) {
 
   const onClick = () => {
     const {gatewayID} = props;
-    MagmaV1API.postNetworksByNetworkIdGatewaysByGatewayIdCommandReboot({
-      networkId: nullthrows(params.networkId),
-      gatewayId: gatewayID,
-    })
-      .then(_resp => {
+    MagmaAPI.commands
+      .networksNetworkIdGatewaysGatewayIdCommandRebootPost({
+        networkId: nullthrows(params.networkId),
+        gatewayId: gatewayID,
+      })
+      .then(() => {
         enqueueSnackbar('Successfully initiated reboot', {variant: 'success'});
         setShowCheck(true);
         setTimeout(() => setShowCheck(false), 5000);
       })
       .catch(error =>
-        enqueueSnackbar('Reboot failed: ' + error.response.data.message, {
+        enqueueSnackbar('Reboot failed: ' + getErrorMessage(error), {
```

Comment:
Was there a preference of the backtick construct over the + operation?
If so, also further below.

Reply:
That is not black and white. If it is more than 2 parts that should be joined or if one of the parts is a not a string I would prefer a formatting string.  But for a simple concatenation like this I feel like a formatting is overkill.